### DISCUSSION TITLE
Limit Number Of Simultaneous Sessions (CP)

### DISCUSSION THREAD

**Discussion by barnabas-szekeres** [statamic/cms#5245] [Q&A]
Hi All,

Is there a possibility to limit the number of logged in sessions per user? I would like to limit the CP admins to one browser session. I guess it's only possible with custom AuthGuard but maybe there is an other way. :)

**Comment by edalzell** [comment]
Why?

**Reply by barnabas-szekeres** [reply]
Because of "security reasons". We had a security audit on our Statamic based project and we plan to solve most of the findings. Of course some of the findings not so thoughtful... :(

However we figured it out how can we manage this:

* Make an EventListener to `Illuminate\Auth\Events\Login` event with the followings
* Check the previous user session id and if it is exist then destroy that session
* Store the current session id for the user
* Profit

Maybe this basically no related to Statamic, rather than Laravel Auth.
